vm/src/builtins.rs

```diff
@@ -440,9 +449,17 @@ mod decl {
         };
 
         if candidates.is_empty() {
-            let default = args.get_optional_kwarg("default");
             return default
                 .ok_or_else(|| vm.new_value_error("max() arg is an empty sequence".to_owned()));
+        } else {
+            let valid_keyword = vec!["default", "key"];
+            for (k, _) in args.kwargs.iter() {
+                if !valid_keyword.contains(&k.as_str()) {
+                    return Err(vm.new_type_error(
+                        format!("'{}' is an invalid keyword argument for max()", k).to_owned(),
```

Comment:
Because `format!` returns `String` object, this `.to_owned()` is redundant. That's what clippy says even though its message is `redundant clone`